Description: javascript libs should be referenced relative to the bash path of a FusionDirectory installation
Author: Mike Gabriel <mike.gabriel@das-netzwerkteam.de>

--- a/core/ihtml/themes/default/headers.tpl
+++ b/core/ihtml/themes/default/headers.tpl
@@ -15,14 +15,14 @@
 
   <link rel="shortcut icon" href="favicon.ico"/>
 
-  <script src="/javascript/prototype/prototype.js" type="text/javascript"></script>
+  <script src="javascript/prototype/prototype.js" type="text/javascript"></script>
   <script src="include/fusiondirectory.js" type="text/javascript"></script>
 {if $usePrototype == 'true'}
-  <script src="/javascript/scriptaculous/scriptaculous.js" type="text/javascript"></script>
-  <script src="/javascript/scriptaculous/builder.js" type="text/javascript"></script>
-  <script src="/javascript/scriptaculous/effects.js" type="text/javascript"></script>
-  <script src="/javascript/scriptaculous/dragdrop.js" type="text/javascript"></script>
-  <script src="/javascript/scriptaculous/controls.js" type="text/javascript"></script>
+  <script src="javascript/scriptaculous/scriptaculous.js" type="text/javascript"></script>
+  <script src="javascript/scriptaculous/builder.js" type="text/javascript"></script>
+  <script src="javascript/scriptaculous/effects.js" type="text/javascript"></script>
+  <script src="javascript/scriptaculous/dragdrop.js" type="text/javascript"></script>
+  <script src="javascript/scriptaculous/controls.js" type="text/javascript"></script>
   <script src="include/pulldown.js" type="text/javascript"></script>
   <script src="include/datepicker.js" type="text/javascript"></script>
 {/if}
--- a/core/ihtml/themes/default/password.tpl
+++ b/core/ihtml/themes/default/password.tpl
@@ -12,7 +12,7 @@
   <style type="text/css">@import url('themes/default/password-style.css');</style>
   <link rel="shortcut icon" href="favicon.ico">
 
-  <script src="/javascript/prototype/prototype.js" type="text/javascript"></script>
+  <script src="javascript/prototype/prototype.js" type="text/javascript"></script>
   <script src="include/fusiondirectory.js" type="text/javascript"></script>
   <script src="include/pwdStrength.js" type="text/javascript"></script>
 </head>
